api/HTMLHyperlinkElementUtils.json

```diff
@@ -0,0 +1,750 @@
+{
+  "api": {
+    "HTMLHyperlinkElementUtils": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLHyperlinkElementUtils",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true,
+            "notes": "Starting in Chrome 52, the members of this interface were moved to <a href='https://developer.mozilla.org/docs/Web/API/URL'>URL</a>"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "22",
+            "notes": [
+              "This mixin was called <code>URLUtils</code> before Firefox 45, and was also implemented to other by other interfaces, like <a href='https://developer.mozilla.org/docs/Web/API/Location'><code>Location</code></a>. From Firefox 45, the other interfaces implement their own version of the properties and methods they need.",
+              "Firefox was a bug whereby single conteined in URLs are escaped when accessed via URL APIs (<a href='https://bugzil.la/1386683'>bug 1386683</a>). This has been as of Firefox 57."
+            ]
+          },
+          "firefox_android": {
+            "version_added": "22",
+            "notes": [
+              "This mixin was called <code>URLUtils</code> before Firefox 45, and was also implemented to other by other interfaces, like <a href='https://developer.mozilla.org/docs/Web/API/Location'><code>Location</code></a>. From Firefox 45, the other interfaces implement their own version of the properties and methods they need.",
+              "Firefox was a bug whereby single conteined in URLs are escaped when accessed via URL APIs (<a href='https://bugzil.la/1386683'>bug 1386683</a>). This has been as of Firefox 57."
```

Comment:
Should be "Firefox has a bug whereby single quotes contained in URLs are escaped..."